modules/ROOT/pages/create-an-api-configure-service-mesh.adoc

```diff
@@ -4,9 +4,30 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: service mesh, microservices, downloading, installing
 
-After you provision the adapter, you must create an API to bind to the service. 
+After you provision the adapter, you must create an API in Anypoint Platform to bind to the service. 
 
-To create an API, perform the following steps:
+== Methods for Creating an API
```

Comment:
Priya-  my understanding based on our last conversation you will have two separate navigation flows 

1. Create an API using Anypoint Platforrm
This will cover what you have mentioned here plus  include this section - " Bind the APIs created using Anypoint Platform"

2. Create an API using service mesh auto.-creation 
This will cover what you have mentioned here plus  include this section -"Bind the APIs created using Anypoint Service Mesh CLI"

Currently, you have binding as a completely different section and missing context. There are two types of binding flows and you get to choose one based on how API is created. so it's really important these two sections are section explained together.

Reply:
Fixed!